contentcuration/contentcuration/static/js/edit_channel/uploader/components/AnswersEditor/__snapshots__/AnswersEditor.spec.js.snap

```diff
@@ -4,31 +4,29 @@ exports[`AnswersEditor for a multiple selection question toolbar contains move u
 <div class="layout justify-end toolbar" data-test="toolbar">
```

Comment:
I'll remove these snapshot specs soon as updates are painful. Just letting know - this was discussed in another PR.

Reply:
Nice! I wasn't sure why they updated, but the changes didn't seem bad.